src/crimson/os/seastore/seastore.h

```diff
@@ -310,8 +327,35 @@ class SeaStore final : public FuturizedStore {
     std::map<std::string, ceph::bufferlist>&& kvs);
 
   boost::intrusive_ptr<SeastoreCollection> _get_collection(const coll_t& cid);
+
+  static constexpr auto LAT_MAX = static_cast<std::size_t>(lat_t::MAX);
+  struct count_lat_t {
+    int64_t num = 0;
+    std::chrono::steady_clock::duration duration_ms =
+                               std::chrono::steady_clock::duration::zero();
+  };
+  struct {
+    std::array<count_lat_t, LAT_MAX> seastore_op_lat;
```

Comment:
why have `seastore` for the prefix of the variable name? i think we can drop it. as we are working only on SeaStore here, so no need to emphasis it or to differentiate it from other store.